core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.ts

```diff
@@ -100,7 +100,12 @@ angular.module('oppia').component('translationOpportunities', {
           },
           controller: 'TranslationModalController'
         }).result.then(function() {
+          ctrl.resetCursorStyle();
+          // This function call is used to reset cursor style to normal
+          // if copy mode is left on and the translation modal is closed down
+          // or the Cancel button is pressed.
```

Comment:
```suggestion
          // Reset cursor style in case the translation modal is closed while copy mode is active.
```

Prefer writing in third-person here, e.g. "This function call is used to" is not necessary. Simplified the comment somewhat.

Reply:
Reworked the comment. It should be more understandable now.